libs/eavmlib/src/network.erl

```diff
@@ -62,6 +62,11 @@
 -type sta_config() :: {sta, [sta_config_property()]}.
 
 -type mac() :: binary().
+-type ap_channel_cfg() :: {ap_channel, 1..14}.
```

Comment:
I added all global 5Ghz channels, I broke them down into bandwith grouped channels, since most devices won't support the faster channels it should aid in selecting appropriate channels based on the bandwith support of the device, once such devices are available for purchase.